README.md

```diff
@@ -14,6 +14,7 @@
     <a href="https://github.com/qdrant/qdrant-client/blob/master/LICENSE"><img src="https://img.shields.io/badge/License-Apache%202.0-success" alt="Apache 2.0 License"></a>
     <a href="https://qdrant.to/discord"><img src="https://img.shields.io/badge/Discord-Qdrant-5865F2.svg?logo=discord" alt="Discord"></a>
     <a href="https://qdrant.to/roadmap"><img src="https://img.shields.io/badge/Roadmap-2023-bc1439.svg" alt="Roadmap 2023"></a>
+    <a href="https://www.npmjs.com/package/@qdrant/js-client-rest"><img src="https://img.shields.io/npm/dm/@qdrant/js-client-rest" alt="Monthly download stats"></a>
```

Comment:
Hi! Thanks for your proposal! I have a question regarding this badge. We introduced also grpc api. Can we calculate sum of rest and grpc?

Reply:
Unfortunately, not. We'll have to show two badges - which doesn't sound like a good idea. 

Plus, given that gprc downloads are relatively low (371 downloads/month), I think it's okay to leave it for now.

Reply:
Unfortunately, not. We'll have to show two badges - which doesn't sound like a good idea. 

Plus, given that gprc downloads are relatively low (371 downloads/month), I think it's okay to leave it for now.

Reply:
I just deployed a small script on Runkit to fix this https://img.shields.io/endpoint?url=https://runkit.io/kshivendu/npm-download-stats/1.0.0?packages=@qdrant/js-client-grpc,@qdrant/js-client-rest :)

Reply:
![limited](https://github.com/qdrant/qdrant-js/assets/47217984/b2e02d21-c250-46bc-ada0-21218c1b2370)

Well looks like runkit frequently rate limits shields.io